test/javascripts/widgets/hamburger-menu-test.js.es6

```diff
@@ -144,6 +144,34 @@ widgetTest("top categories - anonymous", {
   }
 });
 
+widgetTest("top categories - allow_uncategorized_topics", {
+  template: '{{mount-widget widget="hamburger-menu"}}',
+  anonymous: true,
+
+  beforeEach() {
+    this.siteSettings.allow_uncategorized_topics = false;
+    this.siteSettings.header_dropdown_category_count = 8;
+    maxCategoriesToDisplay = this.siteSettings.header_dropdown_category_count;
+    categoriesByCount = this.site
+      .get("categoriesByCount")
+      .filter(c => c.name !== "uncategorized");
+  },
+
+  test(assert) {
+    const count = categoriesByCount.length;
+    const maximum =
+      count <= maxCategoriesToDisplay ? count : maxCategoriesToDisplay;
```

Comment:
Do we need this conditional here if we're dealing with text fixtures?

Reply:
Nope. I simplified both this test and the previous one which is testing when `Uncategorized` is allowed. Thanks!